R/tools.R

```diff
@@ -140,10 +158,21 @@ rhino_style <- function() {
 #' }
 #' @export
 format_r <- function(paths) {
+  if (!box.linters::is_treesitter_installed()) {
+    cli::cli_abort(
```

Comment:
Please change to warning. We should still allow using `format_r` in projects with R <= 4.3.